pkg/terraform/workspace.go

```diff
@@ -111,16 +121,16 @@ func (w *Workspace) ApplyAsync(callback CallbackFn) error {
 		cmd.SetEnv(append(os.Environ(), w.env...))
 		cmd.SetDir(w.dir)
 		out, err := cmd.CombinedOutput()
+		if err != nil {
```

Comment:
I think we are converting a possible error from CombinedOutput to the format of the Apply error before passing it to the callback function. This seems like an improvement to the logs.

Reply:
Hi @sergenyalcin,
Yes, correct. In the synchronous invocations of these commands, we are already transforming the errors so that we do not accidentally leak sensitive info:
https://github.com/upbound/upjet/blob/aeb5f15054f18d0eff52a75e71ec0f562fb79ffb/pkg/terraform/workspace.go#L144

But looks like currently this is not the case in asynchronous invocations.